mundane-csv/src/main/scala/com/ambiata/mundane/csv/SimpleCsv.scala

```diff
@@ -28,14 +28,23 @@ case class SimpleCsv(delimiter: Char, quoteCharacter: Option[Char], escapeCharac
 
   def parse(line: String): String \/ List[String] =
     \/.fromTryCatchNonFatal {
-      if (line.isEmpty) List("")
+      if (line.isEmpty) emptyLineDefault
       else parser.parse(line).toList
```

Comment:
Is the use of `JavaConversions` ok? i thought we were only suppose to use `JavaConverters`.